.travis.yml

```diff
@@ -48,3 +48,10 @@ script:
   # Test pipeline execution: local, conda envs:
   - bash tests/integration/execution/snakemake_local_run_conda_environments.sh
   - md5sum --check tests/integration/expected_output.md5
+  # Test parse.py script
+  - >
+    python scripts/parse.py
+    --indir tests/unit/combine_results/motif_HNRNPF_820 tests/unit/combine_results/motif_HNRNPF_821 tests/unit/combine_results/motif_HNRNPF_822 tests/unit/combine_results/motif_HNRNPF_823 tests/unit/combine_results/motif_HNRNPF_824
+    --filename posterior_sites
+    --outfile tests/unit/combine_results/combined_MotEvo_results.tsv
+  - md5sum --check tests/unit/combine_results/expected_output.md5 
```

Comment:
This is OK, but please put it before the integration test. As a good practice: All unit tests should be executed and passed before the integration test starts.